src/main/webapp/app/pages/teamPage/TeamPage.tsx

```diff
@@ -73,6 +73,13 @@ export const TeamPage = () => {
       faculty: false,
       showCOI: true,
     },
+    {
+      firstName: 'Benjamin',
+      lastName: 'Preiser',
+      title: [TITLE.BSC],
+      faculty: false,
+      showCOI: true,
```

Comment:
@calvinlu3 let's set this to false for now until the update we gonna do for all members.